utils/extraction.py

```diff
@@ -1,15 +1,18 @@
-from typing import Tuple, Optional, Union, cast, Any, TypedDict, List, Type
+from typing import Tuple, Optional, Union, cast, Any, TypedDict, List, Type, Dict
```

Comment:
```suggestion
from typing import (
    Tuple,
    Optional,
    Union,
    cast,
     Any,
     TypedDict,
     List,
     Type,
     Dict,
)
```

Reply:
Hmm. good point. I expected `black` to do these kind of formatting. Seems like it does not do so. will fix.